.github/workflows/goreleaser.yaml

```diff
@@ -76,3 +76,35 @@ jobs:
     with:
       base64-subjects: "${{ needs.goreleaser.outputs.hashes }}"
       upload-assets: true # upload to a new release
+
+  verification:
+    needs: [provenance]
+    runs-on: ubuntu-latest
+    permissions: read-all
+    steps:
+      # Note: this will be replaced with the GHA in the future.
+      - name: Install the verifier
+        uses: slsa-framework/slsa-verifier/actions/installer@v2.1.0
+
+      - name: Download the artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: "${{ needs.provenance.outputs.go-binary-name }}.intoto.jsonl"
+
+      - name: Download the artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: ${{ needs.provenance.outputs.go-binary-name }}
+
+      - name: Verify assets
+        env:
+          ARTIFACT: ${{ needs.provenance.outputs.go-binary-name }}
+          ATT_FILE_NAME: "${{ needs.provenance.outputs.go-binary-name }}.intoto.jsonl"
+        run: |
+          set -euo pipefail
+              
+          echo "Verifying $ARTIFACT using $ATT_FILE_NAME"
```

Comment:
you need to use the list of artifacts from goreleaser and verify them all

Reply:
I just updated accordingly, can you please take a look?